scripts/test-sim-backwards.sh

```diff
@@ -6,6 +6,8 @@ TMP=$(mktemp -d -t tmp.XXXXXXXXXX)
 export STORJ_NETWORK_DIR=$TMP
 cleanup(){
     rm -rf "$STORJ_NETWORK_DIR"
+    git worktree remove -f "$RELEASE_DIR"
+    git worktree remove -f "$BRANCH_DIR"
```

Comment:
does this work for you, i kept getting errors trying to add worktree remove?

Reply:
interesting, this does work for me i'm using `git version 2.22.0` fwiw

Reply:
what does the error look like?

Reply:
at the end when cleanup happens i get an error like this 

```fatal: '/var/folders/s_/n4czty1x4tj297495m66bcsc0000gp/T/tmp.XXXXXXXXXX.Ybi0nSkW/release' is not a working tree```

Reply:
moving `rm -rf "$STORJ_NETWORK_DIR"` after the worktree remove, fixed it